### ISSUE TITLE
Multiple cores on Windows yields fork error "AssertionError: __main__"

### ISSUE THREAD

**Issue by maphew** [saimn/sigal#214]
On my machine with Win7-x64 and 32bit Python 2.7.8 sigal only works if I restrict to a single core:

```
[py27] P:\web_staging\gallery
> sigal build -n 1
Sorting albums  [####################################]  100%
Sorting media  [####################################]  100%
Collecting files  [####################################]  100%
Processing files  [####################################]  1/1C:\Python27\ArcGIS10.3\lib\site-packages\PIL\Image.py:2215: DecompressionBombWarning: Image size (92340000 pixels) exceeds limit of 89478485 pixels, could be decompression bomb DOS attack.
  DecompressionBombWarning)

C:\Python27\ArcGIS10.3\lib\site-packages\sigal\image.py:191: UserWarning: Pillow 3.0.0 is broken with EXIF data, consider using another version if you want to use EXIF data.
  warnings.warn('Pillow 3.0.0 is broken with EXIF data, consider using '
Done.
Processed 1 images and 0 videos in 7.48 seconds.
```

Without `-n 1`:
```
[py27] P:\web_staging\gallery
> sigal build pictures
Sorting albums  [####################################]  100%
Sorting media  [####################################]  100%
Collecting files  [####################################]  100%
Processing files  [------------------------------------]  0/1Traceback (most recent call last):
  File "<string>", line 1, in <module>
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "C:\Python27\ArcGIS10.3\lib\multiprocessing\forking.py", line 380, in main
  File "C:\Python27\ArcGIS10.3\lib\multiprocessing\forking.py", line 380, in main
    prepare(preparation_data)
  File "C:\Python27\ArcGIS10.3\lib\multiprocessing\forking.py", line 488, in prepare
    prepare(preparation_data)
  File "C:\Python27\ArcGIS10.3\lib\multiprocessing\forking.py", line 488, in prepare
    assert main_name not in sys.modules, main_name
AssertionError: __main__
assert main_name not in sys.modules, main_name
...
```
...which repeats forever. Pastebin of `sigal build -fd`: http://pastebin.com/pj5AypHi

Don't know if it's relevant for Sigal, but I found a note https://github.com/patrickfuller/imolecule/issues/11 indicating this can come from using relative imports.

**Comment by saimn** [comment]
From http://stackoverflow.com/a/19798117/4728722 it seems it could be a Python bug, fixed with Python  2.7.11. Which version are you using ?

**Comment by saimn** [comment]
Please reopen and give more details if you still have this issue.
